lib/data/v2/docs.js

```diff
@@ -115,6 +115,11 @@ export default [
             name: 'PHP (@now/php)',
             href: '/docs/v2/deployments/official-builders/php-now-php'
           },
+          {
+            name: 'Wordpress (@now/wordpress)',
+            href:
+              '/docs/v2/deployments/official-builders/wordpress-now-wordpress'
```

Comment:
This format is, unfortunately, a remnant of an old automatic URL formatter and creator. Let's go with the following to get it right in the future:
```suggestion
              '/docs/v2/deployments/official-builders/now-wordpress'
```
And let's update the file-name/URL everywhere else too.

Reply:
Gotcha!